src/App.jsx

```diff
@@ -2,16 +2,26 @@
 import React from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import categoriesFromServer from './api/categories';
-// import productsFromServer from './api/products';
-
-// const products = productsFromServer.map((product) => {
-//   const category = null; // find by product.categoryId
-//   const user = null; // find by category.ownerId
-
-//   return null;
-// });
+import usersFromServer from './api/users';
+import categoriesFromServer from './api/categories';
+import productsFromServer from './api/products';
+
+const products = productsFromServer.map((product) => {
+  const categories = categoriesFromServer
+    .filter(category => category.id === product.categoryId);
+  const user = usersFromServer
+    .find((u) => {
+      const [ownerId] = categoriesFromServer;
```

Comment:
you take always the first category from this array, you should take ownerId from category (line 10, that you found before)